README.md

```diff
@@ -100,6 +100,7 @@ rake test
 To run a single test, use e.g. `ruby test/client_test.rb`.
 
 Coverage is automatically run as part of the tests. After running tests, check `coverage/index.html` for the coverage report.
+Coverage does not work with jruby.
```

Comment:
This should be expanded upon, as this makes it sound like it's flat out broken. The coverage is inaccurate with jruby unless you set `JRUBY_OPTS="--debug"` in the environment. It seems to work then, but it still differs from coverage on MRI.